Lib/fontTools/ufoLib/__init__.py

```diff
@@ -1325,7 +1325,7 @@ def writeLayerContents(self, layerOrder=None, validate=None):
 				if layerName is None:
 					layerName = DEFAULT_LAYER_NAME
 				else:
-					layerName = tounicode(layerName)
+					layerName = tostr(layerName)
```

Comment:
you can just remove the `tostr` (or `tounicode` which are the same on py3) altogether here. Doesn't make sense to continue supporting passing in bytes strings for layer names. The `layerOrder` argument is a `Optional[List[str]]`, so the decode is unnecessary.

Reply:
Ha, I was wondering why the test case has a byte name.